ql/src/java/org/apache/hadoop/hive/ql/ddl/table/create/CreateTableOperation.java

```diff
@@ -99,7 +99,8 @@ public int execute() throws HiveException {
       createTableNonReplaceMode(tbl);
     }
 
-    DDLUtils.addIfAbsentByName(new WriteEntity(tbl, WriteEntity.WriteType.DDL_NO_LOCK), context);
+      DDLUtils.addIfAbsentByName(new WriteEntity(tbl, WriteEntity.WriteType.DDL_NO_LOCK), context);
```

Comment:
what changed here, extra space?

Reply:
Removed populating outputs in CreateTableOperation from the previous commit and retained it only in SemanticAnalyze. So, when removing the previous code, extra space has crept in. 
Will remove the extra line and space.